app/src/main/java/ch/epfl/skysync/components/FlightsList.kt

```diff
@@ -63,7 +63,7 @@ fun FlightsList(
           }
     } else {
       // Display the flights in a LazyColumn if the list is not empty
-      LazyColumn(modifier = Modifier.testTag("HomeLazyList")) {
+      LazyColumn(modifier = Modifier.testTag("HomeLazyList").padding(bottom = 64.dp)) {

```

Comment:
You should put the padding on the column and you should use the paddingValues that is given as parameter. It is the padding of the Scaffold.